index.dd

```diff
@@ -302,7 +303,7 @@ void main()
 
 $(P The $(D @safe), $(D @trusted), and $(D @system) function
 attributes allow the programmer to best decide the safety-efficiency
-tradeoffs of an application, and have the compiler check for
+tradeoffs of an application, and have the compiler guarantee
```

Comment:
I don't think we're yet at the point where we can guarantee these things. There's a few holes to close yet.